### PR TITLE
Fix mistake with License in README

### PR DESCRIPTION
**Pull request by jwmatthews**

### CODE REVIEW

**Review by savitharaghunathan — APPROVED** [review]
/lgtm

File: package.json [hunk 1]
```diff
@@ -319,11 +314,6 @@
                     "when": "view == rhamtExplorerView && viewItem =~ /issue/i",
                     "group": "rhamt@3"
                 },
```
Comment (savitharaghunathan) [inline package.json:316]: John, are these changes removing references to MTA plugin actions?

**Comment by jwmatthews** [comment]
Issue with rebase will close this shortly and resubmit

**Pull request closed without merging**